src/main/resources/org/jenkinsci/plugins/runselector/selectors/DisplayNameRunSelector/help-runDisplayName.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Select the run based on its display name.
```

Comment:
Needs much more info since this selector is considered as unsafe option. Makes sense to explicitly mention that it takes the first matching job and iterates from newest to oldest ones